.github/scripts/discord.sh

```diff
@@ -29,30 +28,41 @@ if [ "$AUTHOR_NAME" == "$COMMITTER_NAME" ]; then
 else
   CREDITS="$AUTHOR_NAME authored & $COMMITTER_NAME committed"
 fi
-
 LINEBREAK=""
 if [ -n "$COMMIT_MESSAGE" ]; then
   LINEBREAK="\n"
 fi
-ADDITIONAL_INFORMATION="$LINEBREAK${COMMIT_MESSAGE//$\\n/ }$LINEBREAK\\n$CREDITS"
+COMMIT_MESSAGE="${COMMIT_MESSAGE//$\\n/ }"
+COMMIT_MESSAGE="${COMMIT_MESSAGE//…/...}"
+ADDITIONAL_INFORMATION="$LINEBREAK$COMMIT_MESSAGE$LINEBREAK\\n$CREDITS"
 
-if [ "$RELEASE" = "release" ]; then
-  RELEASE_NAME="RELEASE"
-  RELEASE_DOWNLOAD_URL="https://github.com/BetonQuest/BetonQuest/releases"
-  RELEASE_DOWNLOAD_URL_DIRECT="https://github.com/BetonQuest/BetonQuest/releases/download/v$VERSION_KEY/BetonQuest.jar"
-  RELEASE_COMMIT_ICON_SUCCESS=$COMMIT_ICON_RELEASE
-else
-  RELEASE_NAME="Dev-Build"
-  RELEASE_DOWNLOAD_URL="https://dev.betonquest.org"
-  RELEASE_DOWNLOAD_URL_DIRECT="https://dev.betonquest.org/api/v1/builds/download/$VERSION_KEY/$VERSION_NUMBER/BetonQuest.jar"
-  RELEASE_COMMIT_ICON_SUCCESS=$COMMIT_ICON_SUCCESS
-fi
+echo "[Webhook]: Preparing type related variables...";
+case "$VERSION_TYPE" in
+  "release" )
+    RELEASE_NAME="Release-Build"
+    RELEASE_DOWNLOAD_URL="https://github.com/BetonQuest/BetonQuest/releases"
+    RELEASE_COMMIT_ICON_SUCCESS=$COMMIT_ICON_RELEASE
+    ;;
+
+  "development" )
+    RELEASE_NAME="Dev-Build"
+    RELEASE_DOWNLOAD_URL="https://betonquest.org/nexus/service/rest/repository/browse/betonquest/org/betonquest/betonquest/2.0.0-SNAPSHOT/"
```

Comment:
This is only a temporary url, the new one in the docs will be places here soon

Reply:
This is fine